.github/dependabot.yml

```diff
@@ -7,3 +7,17 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "daily"
+  - package-ecosystem: "npm"
+    directory: "/website/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "gitsubmodule"
+    directory: "/" # Location of .gitmodules
+    schedule:
+      interval: "daily"
```

Comment:
Will this try to update the submodules to `master`?

Reply:
@kevinbarabash Yes, it'll.